scratch/simulator/README.md

```diff
@@ -15,3 +21,31 @@ TO install GLEW:
 Then install OpenGL with `sudo apt install libgl1-mesa-dev`.
 
 See test.toml for an example level file.
+
+Run make:
+```
+make
+```
+Change directories to the simulator folder:
+```
+cd scratch/simulator
+```
+Make a top level bin folder:
+```
+mkdir bin
+```
+Run make:
+```
+make
+```
+Run the simulator:
+```
+../../bin/simulator test.toml
+```
+To move around the simulator, either scroll in or out; or click and drag to move the screen.
+
+To edit the obstacles, edit the coordinates in test.toml.
+
+To quit, either press escape or x-out.
+
+Whenver you make edits, run `make` and then run the simulator to view changes.
```

Comment:
toml is a missing dependency when I tried to build it. Add it to the instructions and also, please add build instructions.